defaults/main/exec.yml

```diff
@@ -0,0 +1,16 @@
+---
+collectd_plugin_exec_exec: []
+# collectd_plugin_exec_exec:
+# - user: collectd
+#   group: collectd
+#   exec: "my-prog"
+#   args: "arg0" arg1"
+
+collectd_plugin_exec_notification: []
+# collectd_plugin_exec_notification:
+# - user: collectd
+#   group: collectd
+#   notification_exec: "my-notifier"
```

Comment:
Should you also provide the `args` parameter example here?

Reply:
The args example needs to be something different.